test_files/basic.untyped.js

```diff
@@ -0,0 +1,17 @@
+/** @return { ?} */ // This test is just a random collection of typed code, to
+// ensure the output is all with {?} annotations.
+function func(/** ? */ arg1) {
```

Comment:
This had failed to compile before, but rechecking now I think it's due to the comment in the middle, which is issue #18.